codec/src/test/resources/io/netty/handler/codec/xml/sample-04.xml

```diff
@@ -457,14 +457,14 @@
       </plugin>
       <plugin>
         <artifactId>maven-deploy-plugin</artifactId>
-        <version>2.7</version>
+        <version>2.8.2</version>
         <configuration>
           <retryFailedDeploymentCount>10</retryFailedDeploymentCount>
         </configuration>
       </plugin>
       <plugin>
         <artifactId>maven-release-plugin</artifactId>
-        <version>2.4.1</version>
+        <version>2.5.3</version>
```

Comment:
Maybe this one?
http://stackoverflow.com/questions/20912299/using-maven-release-plugin-with-git-1-8-5

It seemed to be an old bug at version 2.4.2